tapchannel/auf_leaf_signer_test.go

```diff
@@ -42,6 +42,51 @@ var (
 	testTimeout = time.Second
 )
 
+// RandCommitmentKeyRing generates a commitment key ring used to derive HTLC
+// signatures.
+func RandCommitmentKeyRing(t *testing.T) lnwallet.CommitmentKeyRing {
```

Comment:
This could be useful in other tests as well. Should we move it to `internal/test/helpers.go`?